tests/Microsoft.Identity.Web.Test/SignedAssertionFilePathCredentialsLoader.cs

```diff
@@ -3,81 +3,76 @@
 
 using System;
 using System.IO;
-using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Identity.Abstractions;
-using Microsoft.Identity.Web.Test.Common;
 using Microsoft.IdentityModel.JsonWebTokens;
 using Xunit;
 
 namespace Microsoft.Identity.Web.Tests.Certificateless
 {
     public class SignedAssertionFilePathCredentialsLoaderTests
     {
-        const string filePath = "signedAssertion.txt";
-        const string aksEnvironmentVariableName = "AZURE_FEDERATED_TOKEN_FILE";
-        string token;
-        SignedAssertionFilePathCredentialsLoader signedAssertionFilePathCredentialsLoader = new SignedAssertionFilePathCredentialsLoader(null);
-
+        private const string FilePath = "signedAssertion.txt";
+        private const string FilePath2 = "signedAssertion2.txt";
+        private const string AksEnvironmentVariableName = "AZURE_FEDERATED_TOKEN_FILE";
+        private readonly string _token;
+        private readonly SignedAssertionFilePathCredentialsLoader _signedAssertionFilePathCredentialsLoader = new(null);
 
         public SignedAssertionFilePathCredentialsLoaderTests()
         {
-            JsonWebTokenHandler handler = new JsonWebTokenHandler();
-            token = handler.CreateToken("{}");
+            JsonWebTokenHandler handler = new();
+            _token = handler.CreateToken("{}");
```

Comment:
would we want to write the files too here?

Reply:
I dn't think it's necessary.